bentoml/yatai/web/src/index.ts

```diff
@@ -8,11 +8,6 @@ const port = args[1];
 const base_log_path = args[2];
 const base_url = args[3];
 const prometheus_address = args[4];
-if (!grpcServerAddress || !prometheus_address || !port || !base_log_path) {
```

Comment:
I think we do need this to validate the run options under `package.json` or the given commands under `Makefile`.

Reply:
Thank you for pointing this out. I removed this in err.